workers/default/utils/tasks.py

```diff
@@ -49,6 +49,39 @@ def execute_task(self):
 ForeachTask = celery_app.register_task(ForeachTask())
 
 
+class IfTask(BaseTask):
+    """
+    Run a task list if a condition is met and optionally another one if not.
+
+    TaskParams:
+    -boolean condition: Condition to be checked upon
+    -TaskList do:
+    -TaskList else:
+
+    Preconditions:
+
+    Creates:
+
+    """
+    name = "if"
+
+    def execute_task(self):
+        condition = self.get_param('condition')
+        do = self.get_param('do')
+        group_tasks = []
+        params = self.params.copy()
+        if condition:
```

Comment:
Oder an dieser Stelle die `condition` mit `eval()` testen. Das hätte den Vorteil, dass die Parameter in den vorhergehenden Tasks noch verändert werden können. Dafür müsste aber `params['result']` bei der Evaluierung der Bedingung ebenfalls berücksichtigt werden.